README.md

```diff
@@ -35,9 +49,14 @@ on:
   schedule:
   - cron: '0 0 * * 1'
 
+permissions: {}
+
 jobs:
   ci:
     name: CI
+    permissions:
+      pull-requests: read
+      actions: write
```

Comment:
Are we 100% sure we actually need to define these with a default github actions token? Namely for people using gha-ci in their newly created silverstripe module
- Isn't the default token read only in all scope? So we don't need `pull-requests: read`?
- We're triggering the action via the API - though conditionally behind checkgovernance which check the org in `silverstripe` i.e. the action won't get called by forks

Reply:
Nope, was waiting for the first round of PRs to be merged because I started farting around testing this. There's a reason I'm doing this in two lots. :p